frontend/src/hackathon.jsx

```diff
@@ -0,0 +1,71 @@
+import Axios from 'axios'
+import { Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import  { useState } from 'react';
+
+const Hackathon = () => {
+  const [Hackathonlist, setHackathonlist] = useState([]);
+  
+  const getHackathon = async () => {
+    try {
+      const response = await Axios.get('http://localhost:3307/hackathon');
```

Comment:
use `localhost:3000`

Reply:
Nice to have : Try using the .env in frontend folder to store the apiUrl path to easier change in future.

For example : 
`const apiUrl = import.meta.env.BACKEND_URL;`